### ISSUE TITLE
OpenType

### ISSUE THREAD

**Issue by dpk** [edwardtufte/et-bembo#3]
The lining figures and text figures versions of the font could be rolled into single font files containing both, using OpenType features to select between the two. (The features `lnum` and `onum` in particular.)

Likewise the small-capitals fonts (`smcp` and `c2sc`) and ‘expert set’ (mapped to the correct Unicode codepoints) included in the originals.zip file in the source directory can be merged into the main files of the typeface using the OpenType format.

Then, once the [last few browsers support it,](http://caniuse.com/#feat=font-feature) webfont users could enable these using the standard [`font-feature-settings`](http://clagnut.com/sandbox/css3/) CSS property.

**Comment by ghost** [comment]
This is true. In addition, it would be really great if automatic ligature insertion could be added, since the glyphs are already there, just not called. This could be done in FontForge quite easily. Other than that this is a great, exciting project.

I would personally prefer having an SC font kept in the build for one key use - desktop apps, which mostly can't use true small caps. Though hardly a huge problem for me because I can just use the originals zip.

**Comment by dpk** [comment]
Oh yeah, ligatures too. I didn’t test to see that they aren’t substituted by default.

**Comment by dpk** [comment]
I have begun to do this with FontForge in my personal fork at [dpk/et-book](https://github.com/dpk/et-book).

**Comment by dpk** [comment]
I’ve now done this completely with one exception: I can’t work out how to get the `frac` feature to work to enable automatic creation of fractions from strings such as “1/2”. This involves some complicated contextual substitution tables which I don’t know how to set up.

**Comment by adrientetar** [comment]
http://www.opentypecookbook.com/

**Comment by KrasnayaPloshchad** [comment]
Recently LibreOffice made some improves to make them available on document.
https://wiki.documentfoundation.org/ReleaseNotes/5.3
https://ask.libreoffice.org/en/question/82052/opentype-features-now-enabled-documentation/

**Comment by KrasnayaPloshchad** [comment]
Recently LibreOffice made some improved to make them available on document.
https://wiki.documentfoundation.org/ReleaseNotes/5.3

**Comment by KrasnayaPloshchad** [comment]
Recently LibreOffice made some improved to make them available on document.
https://wiki.documentfoundation.org/ReleaseNotes/5.3

**Comment by gheoan** [comment]
Related: https://github.com/edwardtufte/tufte-css/issues/99.

**Comment by KrasnayaPloshchad** [comment]
We should consider for TeX engine as XeTeX which has an entry to access typographic feature.
